dataloader.go

```diff
@@ -208,11 +208,6 @@ func (l *Loader) Load(key string) Thunk {
 		l.curBatcher = l.newBatcher()
 		// start the current batcher batch function
 		go func(b *batcher) {
```

Comment:
Now that we have have removed the defer from this function, we no longer need it. I would remove the entire function and instead call `go l.curBatcher.batch()` directly.